bridetalk_project/src/components/pages/reset-password/ResetPage.js

```diff
@@ -0,0 +1,45 @@
+import {
+  Headline1,
+  PrimaryButton,
+  EnterEmail,
+  CancelButton,
+} from "../../atoms";
+import "./resetPage.css";
+
+
+
+export const ResetPage = () => {
```

Comment:
ResetPasswordPage maybe?
So the classnames, from reset- to reset-password- ?
ResetPage can mean anything